cmd/oas-cache/oas-cache.go

```diff
@@ -0,0 +1,73 @@
+// CLI utility that builds cache for OAS file to reduce init time.
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/go-openapi/loads"
+	"github.com/go-openapi/spec"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/validate"
+)
+
+const help = `
+Usage:
+	oas-cache [SPEC_FILE] [CACHE_FILE]
```

Comment:
Maybe we should name it like "expander" because in fact it just expands the schema. And we use expanded schema as a "cache", so it is just in our context.

Reply:
Sounds good. Done.